scripts/publish.js

```diff
@@ -315,7 +339,13 @@ const Publish = {
                     // when called with `--json`, you get a JSON error.
                     // this could also be handled in a catch() chained to the `exec` promise
                     if ('stdout' in err) {
-                      /** @type {{error: {code: string, summary: string, detail: string}}} */
+                      /** @type {{
+  error: {
+    code: string
+    summary: string
+    detail: string
+  }
+}} */
```

Comment:
prettier went bonkers?

Reply:
hmm, changes to this file should not be in this PR.  I'll pull that out